x-pack/plugin/esql-core/src/main/java/org/elasticsearch/xpack/esql/core/type/DataType.java

```diff
@@ -267,8 +267,8 @@ public enum DataType {
      */
     // 8.15.2-SNAPSHOT is 15 bytes, most are shorter, some can be longer
     VERSION(builder().esType("version").estimatedSize(15).docValues()),
-    OBJECT(builder().esType("object").unknownSize()),
-    SOURCE(builder().esType(SourceFieldMapper.NAME).unknownSize()),
+    OBJECT(builder().esType("object").estimatedSize(1024)),
+    SOURCE(builder().esType(SourceFieldMapper.NAME).estimatedSize(1024)),
```

Comment:
I wonder if it is worth having even higher estimate for source?
I imagine logs source documents could easily reach 10k